hs/WebParsing/TimeTableParser.hs

```diff
@@ -198,4 +200,7 @@ parseTT = do
     rsp <- simpleHTTP (getRequest timetableUrl)
     body <- getResponseBody rsp
     let depts = getDeptList $ parseTags body
-    mapM_ getDeptTimetable depts
+
+    runSqlite databasePath $ do
+        runMigration migrateAll
```

Comment:
So this seems to be the substantial change. Is a migration even necessary here if we're guaranteed to have performed it earlier?